src/ocm/components/clusterConfiguration/staticIp/data/formDataFromInfraEnvField.ts

```diff
@@ -145,7 +155,8 @@ const getFormViewHost = (
     },
   };
   for (const protocolVersion of getShownProtocolVersions(protocolType)) {
-    ret.ips[protocolVersion] = getIpAddress(ethernetInterface, protocolVersion);
+    if (!vlanInterface) ret.ips[protocolVersion] = getIpAddress(ethernetInterface, protocolVersion);
```

Comment:
It's better to avoid the double negation (so putting the `if (vlanInterface)` as the  first condition) as it doesn't complicate the flow